x/concentrated-liquidity/pool_test.go

```diff
@@ -59,6 +60,13 @@ func (s *KeeperTestSuite) TestInitializePool() {
 			creatorAddress: validCreatorAddress,
 			expectedErr:    fmt.Errorf("invalid swap fee. Got %d", invalidSwapFee),
 		},
+		{
+			name:                      "unauthorized quote denom",
+			poolI:                     validPoolI,
+			authorizedDenomsOverwrite: []string{"otherDenom"},
+			creatorAddress:            validCreatorAddress,
+			expectedErr:               types.UnauthorizedQuoteDenomError{Denom: USDC},
```

Comment:
This took me two read throughs to understand. I think we are overwriting all authorized denoms with "otherDenom" which makes USDC no longer authorized rightr? Maybe just make it a boolean that says "isQuoteAssetAuthorized" would make it easier to reason about.

Reply:
I considered this but don't think there is a significant improvement. In the suggested case, we would have to provide `true` flags for many of the test cases where quote assets are authorized. Currently, we simply provide overwrite where we intend it to be unauthorized, minimizing the cognitive overhead from having to parse many flags.

I added a comment to help with readability